groups/tests/test_managers.py

```diff
@@ -0,0 +1,108 @@
+from django.test import TestCase
+from incuna_test_utils.compat import Python2AssertMixin
+
+from . import factories
+from .. import models
+
+
+class TestGroupManager(Python2AssertMixin, TestCase):
+    def test_discussions(self):
+        discussion = factories.DiscussionFactory.create()
+        discussion_two = factories.DiscussionFactory.create()  # Has a different group
```

Comment:
It doesn't massively matter; the method will do the same thing if you have discussions from different groups as it would if they were both the same.  It's just for the reader's information/clarity.  I didn't think adding two more lines of code would be likely to make it any clearer.